### PR TITLE
[20176] Bump version to 2.13.1

### PR DESCRIPTION
**Pull request by MiguelCompany**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/134/)

**Review by cferreiragonz — APPROVED** [review]

File: docs/notes/notes.rst [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
-This minor release includes the following **improvements**:
+This patch release includes the following **improvements**:
 
-* Support for Fast DDS v2.13.0
-* Add new option to configure data representation in publisher
-* Add new option to enable/disable monitor service
-* Add metadata to identify Shapes Demo application
+* Support for Fast DDS v2.13.1
+* Statistics and monitor are enabled by default
```
Comment (JesusPoderoso) [inline docs/notes/notes.rst:7]: Include latest PRs merged
```suggestion
* Statistics and monitor service are enabled by default
* Introduced new UDP transport that intentionally drops a percentage of samples
* Downgrade CMake minimum required version
* Regenerate Shapes types with Fast DDS-Gen v3.2.1 
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/139/)

**Pull request merged**